exit-intent-popup/template.ts

```diff
@@ -22,11 +22,10 @@ export class ExitIntentPopupTemplate implements CampaignTemplateComponent {
     @richText(true)
     header: string = "Header Text";
 
+    @subtitle("Optional rich text field")
     @richText(true)
     subheader: string = "Subheader Text";
 
-    subheaderVisibility: boolean = true;
-
     @title("CTA Text")
     ctaText: string ="Call To Action";
```

Comment:
Could you make this minor update too?
```suggestion
    ctaText: string = "Call To Action";
```

Reply:
Lol How'd that happen? 🙃